packages/lit-dev-content/samples/examples/react-basics/index.html

```diff
@@ -0,0 +1,9 @@
+<!doctype html>
+<head>
+  <script src="https://unpkg.com/react@18/umd/react.development.js"></script>
+  <script src="https://unpkg.com/react-dom@18/umd/react-dom.development.js"></script>
+  <script type="module" src="./index.js"></script>
+</head>
+<body>
+  <section></section>
```

Comment:
React was made before HTML 5 was a thing. They'd probably use main these days. The only preference react carries is that you don't target `body`.

Reply:
As in, there can be multiple 'roots' on a page so the classic `<div id="root">` is not exactly correct just standard. Multiple roots can't share contexts though

Reply:
I'd argue `<div id="root">` is better than `<section>` since by html semantics it's meant to be unique. A different react root can then just be created by providing a different div with a different id like `<div id="root2">`.

Reply:
each `section` is unique yes, which would be appropriate for multiple roots that are unique as well.

if readability is a thing though then sure we can do the `<div>` root stuff, i dont mind much

Reply:
I'm okay with keeping as is.

Just to finish the thought out, if there are multiple `<section>` tags to serve as root of the react app, they'd still need unique ids so they can be selected and passed into `ReactDOM.createRoot()`.

Also, when to use a section over a div seems to be not super straight-forward.
https://stackoverflow.com/questions/6939864/what-is-the-difference-between-section-and-div
I've tended to go with the rule of does this section get its own heading as the MDN doc says https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section though it's not a hard and fast rule and semantically a whole react app could be construed as a "section" of the document.

Reply:
I'd argue the react app _is_ a section of the document. Common practice is to have the minimal amount of your app load and then let react take over. Yes react is what makes the app "go" but it's just one small part of a website.